extensions/sql-database-projects/src/test/testContext.ts

```diff
@@ -22,6 +22,12 @@ export const mockDacFxResult = {
 	report: ''
 };
 
+export const mockSavePublishResult = {
+	success: true,
+	errorMessage: '',
+	report: ''
```

Comment:
what's `report`?

Reply:
Copy-paste hazard.
Removed it for next iteration.